app/redux/MarketSaga.js

```diff
@@ -68,9 +69,12 @@ export function* fetchOpenOrders(set_user_action) {
         const state = yield call([db_api, db_api.exec], 'get_open_orders', [username]);
         yield put(MarketReducer.actions.receiveOpenOrders(state));
 
-       const [account] = yield call(Apis.db_api, 'get_accounts', [username])
-       yield put(MarketReducer.actions.receiveAccount({ account }))
-
+       let [account] = yield call(Apis.db_api, 'get_accounts', [username])
+       if(account) {
+           account = fromJS(account)
+           yield put(MarketReducer.actions.receiveAccount({ account }))
+           yield put(g.actions.receiveAccount({ account }))
```

Comment:
Hm you're right actually. The issue is that fetchOpenOrders is the only place that calls get_accounts with your account when you log in. If we can put that in `loginWatch` in UserSaga for example that would be better, or we can add a special case in `fetchState`.